nest.py

```diff
@@ -43,7 +43,7 @@ class _Helper:
     from_f_to_celsuis = lambda fahrenheit : round((fahrenheit - 32) / 1.8, 1)
     
     # headers for API Calls
-    def get_headers():
+    def get_headers(self):
```

Comment:
Causes an error get_headers is a static method. i forgot to add @staticmethod. 
Fix: remove self and add @staticmethod to method.